infra/pr_helper.py

```diff
@@ -0,0 +1,246 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Adds comments for PR to provide more information for approvers."""
+import base64
+import json
+import os
+import subprocess
+
+import requests
+import yaml
+
+OWNER = 'google'
+REPO = 'oss-fuzz'
+GITHUB_URL = 'https://github.com/'
+API_URL = 'https://api.github.com'
+BASE_URL = f'{API_URL}/repos/{OWNER}/{REPO}'
+BRANCH = 'master'
+CRITICALITY_SCORE_PATH = '/home/runner/go/bin/criticality_score'
+
+
+def get_criticality_score(repo_url):
+  """Gets the criticality score of the project."""
+  report = subprocess.run([
+      CRITICALITY_SCORE_PATH, '--format', 'json',
+      '-gcp-project-id=clusterfuzz-external', '-depsdev-disable', repo_url
+  ],
+                          capture_output=True,
+                          text=True)
+
+  report_dict = json.loads(report.stdout)
+  return report_dict.get('default_score', 'N/A')
+
+
+def is_known_contributor(content, email):
+  """Checks if the author is in the contact list."""
+  return (email == content.get('primary_contact') or
+          email in content.get('vendor_ccs', []) or
+          email in content.get('auto_ccs', []))
+
+
+def save_env(message, is_ready_for_merge, is_internal=False):
+  """Saves the outputs as environment variables."""
+  with open(os.environ['GITHUB_ENV'], 'a') as github_env:
+    github_env.write(f'MESSAGE={message}\n')
+    github_env.write(f'IS_READY_FOR_MERGE={is_ready_for_merge}\n')
+    github_env.write(f'IS_INTERNAL={is_internal}')
+
+
+def main():
+  """Verifies if a PR is ready for merge."""
+  message = ''
+  is_ready_for_merge = True
+  github = GithubHandler()
+  pr_author = github.get_pr_author()
+
+  # Bypasses PRs of the internal members.
+  if github.is_author_internal_member():
+    save_env(None, None, True)
+    return
+
+  # Gets all modified projects path.
+  projects_path = github.get_projects_path()
+  email_response = github.get_author_email()
+
+  for project_path in projects_path:
+    project_url = f'{GITHUB_URL}/{OWNER}/{REPO}/tree/{BRANCH}/{project_path}'
+    content_dict = github.get_project_yaml(project_path)
+
+    # Gets information for the new integrating project
+    if not content_dict:
+      is_ready_for_merge = False
+      new_project = github.get_integrated_project_info()
+      repo_url = new_project.get('main_repo')
+      if repo_url is not None:
+        message += (f'@{pr_author} is integrating a new project:<br/>'
+                    f'- Main repo: {repo_url}<br/> - Criticality score: '
+                    f'{get_criticality_score(repo_url)}<br/>')
+      else:
+        message += (f'@{pr_author} is integrating a new project, '
+                    'but the `repo_url` is missing. '
+                    'The criticality score cannot be computed.<br/>')
```

Comment:
```
if repo_url is None:
  ...
else:
  ...
```
Since we have both `if` and `else` now, I would suggest flipping their order to make the code easier to understand : )